app/uk/gov/hmrc/gform/ApplicationLoader.scala

```diff
@@ -81,7 +81,7 @@ class ApplicationModule(context: Context) extends BuiltInComponentsFromContext(c
   lazy val save4later =
     new Save4Later(shortLivedCacheModule.shortLivedCache)
 
-  lazy val formService = new FormService(save4later, fileUploadModule.fileUploadService)
+  lazy val formService = new FormService(save4later, fileUploadModule.fileUploadService, configModule)
```

Comment:
I believe this is the first instance of us injection the configModule into a *service*, rather than given the service what it needs from the configModule. Do we want to go this way?